### PR TITLE
Added maintainers column to backend projects

### LINKED ISSUE #5057: [Community] Handbook: Add maintainer column to backend projects table
### Current Behavior
https://layer5.io/community/handbook/repository-overview lists maintainers of front-end projects, but not of backend projects.

### Desired Behavior
Add a maintainer column to backend projects table. 

### Implementation
1. Support multiple maintainers per project.
1. Support a hyperlink to the maintainers community profile.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by ankushbhardwaj408**
**Description**

This PR fixes #5057 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 1faf54ea7d0e7b4b178e933c8120f0a9a94ee01f at: https://653fe3a92f843312ba33995d--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9f9b6566bdd7086bf22af9373a288cf31295094e at: https://654614e7a334a965c8c3c612--layer5.netlify.app

**Comment by abhijeetgauravm** [comment]
Hey @ankushbhardwaj408  Let's discuss this on website call tomorrow at 6:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Review by ritiksaxena124 — COMMENTED** [review]
Merge Conflicts

**Comment by ankushbhardwaj408** [comment]
@ritiksaxena124 Done.

**Comment by l5io** [comment]
🚀 Preview for commit 96045fa78b6e929ff2148c4f2ad0a561a3cc2ba1 at: https://6547d89816594777825ea358--layer5.netlify.app

**Review by iArchitSharma — APPROVED** [review]
LGTM!

**Comment by l5io** [comment]
🚀 Preview for commit b5d621e22c410583763f823b2f853c92f380ab71 at: https://6549222f3350cd1d91c122d7--layer5.netlify.app

**Review by Mohith234 — APPROVED** [review]
Yay 🎈, LGTM @ankushbhardwaj408

**Review by leecalcote — CHANGES_REQUESTED** [review]
Will you please undo any changes not directly related to the issue?

**Comment by ankushbhardwaj408** [comment]
Thank you for your input. I have carefully reviewed the changes, and there are no changes that are not related to the problem, if there are still any please let me know

**Comment by iArchitSharma** [comment]
> @leecalcote Thank you for your input. I have carefully reviewed the changes, and there are no changes that are not related to the problem, if there are still any please let me know

@ankushbhardwaj408 I think Lee is referring to the code formatting here. Can you keep the formatting as it was?

**Comment by ankushbhardwaj408** [comment]
@iArchitSharma Done.

**Comment by l5io** [comment]
🚀 Preview for commit 7905cd7657fe9992130d115795df19b22540fdc0 at: https://6549ffb261fe677640aa10a0--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3425bd883e52e0e536b338a75eabcc01789a61e5 at: https://6552508f9684d803e6ee8fed--layer5.netlify.app

**Comment by iArchitSharma** [comment]
LGTM!

File: src/sections/Community/Handbook/repo-data.js [hunk 1]
```diff
@@ -321,6 +364,8 @@ export const repo_data = [
         project: "Gowrk2",
         image: meshery,
         language: "Golang",
+        maintainers_name: ["Vacant"],
```
[the remark this answers is not in the document]
Comment (ankushbhardwaj408) [reply src/sections/Community/Handbook/repo-data.js:367]: On it.

File: src/sections/Community/Handbook/repo-data.js [hunk 2]
```diff
@@ -380,110 +437,139 @@ export const repo_data = [
...
+        link: ["#"],
         description: "Meshery adapter for management of Kuma.",
         repository: "https://github.com/layer5io/meshery-kuma",
       },
       {
         project: "Meshery Adapter for Consul",
         image: consulLogo,
         language: "Golang, gRPC",
+        maintainers_name: ["Vacant"],
```
[the remark this answers is not in the document]
Comment (ankushbhardwaj408) [reply src/sections/Community/Handbook/repo-data.js:494]: Screenshot 2023-11-14 195116
It is not specified who is the maintainer of which repo

**Comment by leecalcote** [comment]
I wonder what the purpose of the "ADAPTERS MAINTAINERS" section is in your screenshot.

File: src/sections/Community/Handbook/repo-data.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (leecalcote) [reply src/sections/Community/Handbook/repo-data.js:494]: I wonder what the purpose of the "ADAPTERS MAINTAINERS" section is in your screenshot.

[the remark this answers is not in the document]
Comment (ankushbhardwaj408) [reply src/sections/Community/Handbook/repo-data.js:494]: Okay i got it😅

**Comment by l5io** [comment]
🚀 Preview for commit 3cd4c0c453fe71fa80ff9254a507514475b1691a at: https://655390a1f2ce642abf096377--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e8f66895f6b73a4024d4539083489b3e2ccd75a4 at: https://655395d9047e58009b621373--layer5.netlify.app

File: src/sections/Community/Handbook/repo-data.js [hunk 3]
```diff
@@ -41,26 +41,29 @@ export const repo_data = [
...
       },
       {
         project: "Meshery Documentation",
         image: meshery,
         site: "http://docs.meshery.io",
         language: "Jekyll",
-        maintainers_name: ["Vacant"],
-        link: ["#"],
+        maintainers_name: ["Lee Calcote"],
+        link: ["https://layer5.io/community/members/lee-calcote"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:54]:
```suggestion
        maintainers_name: ["Vivek Vishal"],
        link: ["https://layer5.io/community/members/vivek-vishal"],
```

File: src/sections/Community/Handbook/repo-data.js [hunk 4]
```diff
@@ -41,26 +41,29 @@ export const repo_data = [
...
         image: meshery,
         site: "https://github.com/meshery/meshery/tree/master/ui",
         language: "NextJS, React.js, MaterialUI and Billboard.js",
-        maintainers_name: ["Antonette Caldwell","Abhishek Kumar"],
-        link: ["https://layer5.io/community/members/antonette-caldwell","https://layer5.io/community/members/abhishek-kumar"],
+        maintainers_name: ["Antonette Caldwell", "Abhishek Kumar"],
+        link: [
+          "https://layer5.io/community/members/antonette-caldwell",
+          "https://layer5.io/community/members/abhishek-kumar",
+        ],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:66]:
```suggestion
        maintainers_name: ["Antonette Caldwell", "Aabid Sofi"],
        link: [
          "https://layer5.io/community/members/antonette-caldwell",
          "https://layer5.io/community/members/aabid-sofi",
        ],
```

@aabidsofi19

Comment (leecalcote) [reply src/sections/Community/Handbook/repo-data.js:66]: @ankushbhardwaj408 will you PR this change in the respective maintainers.md?

File: src/sections/Community/Handbook/repo-data.js (hunk 3 above)
Comment (leecalcote) [reply src/sections/Community/Handbook/repo-data.js:54]: @ankushbhardwaj408 will you PR this change in the respective maintainers.md?

File: src/sections/Community/Handbook/repo-data.js [hunk 5]
```diff
@@ -108,9 +111,10 @@ export const repo_data = [
         image: patterns,
         site: "https://service-mesh-patterns.github.io/service-mesh-patterns/",
         language: "Gatsby",
-        maintainers_name: ["vacant"],
-        link: ["https://layer5.io/community/members/#"],
-        repository: "https://github.com/service-mesh-patterns/service-mesh-patterns",
+        maintainers_name: ["Lee Calcote"],
+        link: ["https://layer5.io/community/members/lee-calcote"],
+        repository:
+          "https://github.com/service-mesh-patterns/service-mesh-patterns",
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:117]: Please add @nicholasjackson as a maintainer.

File: src/sections/Community/Handbook/repo-data.js [hunk 6]
```diff
@@ -147,7 +152,7 @@ export const repo_data = [
         language: "React.js",
         maintainers_name: ["Antonette Caldwell"],
         link: ["https://layer5.io/community/members/antonette-caldwell"],
-        repository: "https://github.com/layer5io/sistent"
+        repository: "https://github.com/layer5io/sistent",
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:155]: Please add @aabidsofi19 as a maintainer here (and in the maintainers.md in that repo).

Comment (leecalcote) [reply src/sections/Community/Handbook/repo-data.js:155]: Please add @Rexford74, too.

File: src/sections/Community/Handbook/repo-data.js [hunk 7]
```diff
@@ -168,24 +173,28 @@ export const repo_data = [
         project: "Cloud Native Labs",
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:173]: Please change this to Cloud Native Learning Paths (https://layer5.io/learn/learning-paths)

File: src/sections/Community/Handbook/repo-data.js [hunk 8]
```diff
@@ -195,29 +204,31 @@ export const repo_data = [
...
       {
         project: "Cloud Native Playground",
         image: meshery,
         language: "Meshery",
-        description:
-          "Cloud Native Playground for Kubernetes and CNCF projects",
-        repository:
-          "https://github.com/meshery/play",
-      }
+        maintainers_name: ["Nikhil Ladha"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:227]: Please leave maintainer vacant.

File: src/sections/Community/Handbook/repo-data.js [hunk 9]
```diff
@@ -228,37 +239,56 @@ export const repo_data = [
         project: "Meshery",
         image: meshery,
         language: "Golang, gRPC",
-        description: "the cloud native management plane to provide governance and backend system integration. Golang, gRPC, NATS, CloudEvents",
+        maintainers_name: ["Nikhil Ladha", "Aaditya Narayan Subedy"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:242]: Please swap with @MUzairS15 and @nebula-aac

File: src/sections/Community/Handbook/repo-data.js [hunk 10]
```diff
@@ -228,37 +239,56 @@ export const repo_data = [
...
+        description:
+          "Meshery Operator is the multi-cloud native operator and implementation of MeshSync.",
         repository: "https://github.com/meshery/meshery-operator",
       },
       {
         project: "MeshSync",
         image: MeshSync,
         language: "Golang and NATS",
-        description: "MeshSync is a multi-mesh resource discovery application which is also a custom kubernetes controller.",
+        maintainers_name: ["Vacant"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:276]: Please add @theBeginner86

File: src/sections/Community/Handbook/repo-data.js [hunk 11]
```diff
@@ -284,14 +314,20 @@ export const repo_data = [
         project: "Image Hub",
         image: imagehub,
         language: "Vuejs",
-        description: "A sample application written to run on Istio or Consul for exploring WebAssembly modules used as Envoy filters.",
+        maintainers_name: ["Lee Calcote"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:317]: Please swap with @theBeginner86

File: src/sections/Community/Handbook/repo-data.js [hunk 12]
```diff
@@ -339,16 +391,22 @@ export const repo_data = [
...
+        repository: "https://github.com/layer5io/meshery-smp-action",
       },
       {
         project: "Meshery-Perf",
         image: meshery,
         language: "Golang",
-        description: "meshery-perf is a Meshery component used to run performance tests",
-        repository: "https://github.com/meshery/meshery-perf"
-      }
+        maintainers_name: ["Vacant"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:404]: @gyohuangxin?  - - he'll have to confirm...

File: src/sections/Community/Handbook/repo-data.js [hunk 13]
```diff
@@ -359,6 +417,12 @@ export const repo_data = [
         project: "Cloud Native Patterns",
         image: patterns,
         language: "Yaml",
+        maintainers_name: ["Lee Calcote", "Nic Jackson", "Nikhil Ladha"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:420]: Remove Nikhil.

File: src/sections/Community/Handbook/repo-data.js [hunk 14]
```diff
@@ -380,110 +448,142 @@ export const repo_data = [
...
+        maintainers_name: ["Michael Gfeller"],
+        link: ["https://layer5.io/community/members/michael-gfeller"],
         description: "Meshery adapter for management of Network Service Mesh.",
         repository: "https://github.com/layer5io/meshery-nsm",
       },
       {
         project: "Meshery Adapter for Octarine",
         image: meshery,
         language: "Golang, gRPC",
+        maintainers_name: ["Lee Calcote"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:478]: Please move this Adapter altogether. It has been archived.

File: src/sections/Community/Handbook/repo-data.js [hunk 15]
```diff
@@ -380,110 +448,142 @@ export const repo_data = [
...
+        link: ["https://layer5.io/community/members/antonette-caldwell"],
         description: "Meshery adapter for management of AWS App Mesh",
         repository: "https://github.com/layer5io/meshery-app-mesh",
       },
       {
         project: "Meshery Adapter for Tanzu Service Mesh",
         image: meshery,
         language: "Golang, gRPC",
-        description: "Meshery adapter for management of VMware Tanzu Service Mesh",
+        maintainers_name: ["Xin Huang"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:532]: This should be @hexxdump

File: src/sections/Community/Handbook/repo-data.js [hunk 16]
```diff
@@ -380,110 +448,142 @@ export const repo_data = [
...
+        link: ["https://layer5.io/community/members/xin-huang"],
         description: "Meshery adapter for management of NGINX Service Mesh",
         repository: "https://github.com/layer5io/meshery-nginx-sm",
       },
       {
         project: "Meshery Adapter for Cilium Service Mesh",
         image: ciliumLogo,
         language: "Golang, gRPC",
+        maintainers_name: ["Xin Huang"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:551]: Switch to @MUzairS15

File: src/sections/Community/Handbook/repo-data.js [hunk 17]
```diff
@@ -380,110 +448,142 @@ export const repo_data = [
...
+        description:
+          "A common and consistent set of functionality for Meshery Adapters.",
         repository: "https://github.com/layer5io/meshery-adapter-library",
       },
       {
         project: "Meshery Adapter Template",
         image: meshery,
         language: "Golang, gRPC",
-        description: "A GitHub repo used as a template for new Meshery Adapters.",
+        maintainers_name: ["Antonette Caldwell"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:579]: Please leave vacant.

File: src/sections/Community/Handbook/repo-data.js [hunk 18]
```diff
@@ -195,29 +204,31 @@ export const repo_data = [
         project: "Linkerd Service Mesh Workshop",
         image: linkerdLogo,
         language: "Meshery",
+        maintainers_name: ["Hussaina Begum"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:207]: Please leave vacant.

File: src/sections/Community/Handbook/repo-data.js [hunk 19]
```diff
@@ -304,6 +340,18 @@ export const repo_data = [
         project: "Cloud Native Performance",
         image: smp,
         language: "Protobuf",
+        maintainers_name: [
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:343]: Please add @asubedy

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: src/sections/Community/Handbook/repo-data.js (hunk 3 above)
Comment (ankushbhardwaj408) [reply src/sections/Community/Handbook/repo-data.js:54]: @leecalcote Should I add maintainers.md to all repos or just to the specific repo you asked me to

File: src/sections/Community/Handbook/repo-data.js (hunk 12 above)
Comment (ankushbhardwaj408) [reply src/sections/Community/Handbook/repo-data.js:407]: I'll change it after @gyohuangxin confirms.

**Comment by l5io** [comment]
🚀 Preview for commit 0fe13da9b935ee33d4eead3f3cbbdefe826d7cb1 at: https://6554487fd4c7e0572ef28aef--layer5.netlify.app

**Comment by ankushbhardwaj408** [comment]
@leecalcote I've created  PRs to add maintainers.md https://github.com/layer5io/sistent/pull/339 https://github.com/meshery/meshery/pull/9365

**Comment by l5io** [comment]
🚀 Preview for commit 767073cd6c6b3d3af9a9d4358ed95d6c84f9f8aa at: https://65544dfc4bbe7259d05f3357--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3e2eeba9542400c2b08b7fd3a5e2903c971d89b2 at: https://6555965f0030203272b79370--layer5.netlify.app

File: src/sections/Community/Handbook/repo-data.js [hunk 20]
```diff
@@ -368,8 +440,12 @@ export const repo_data = [
...
         image: patterns,
         language: "shell",
-        description: "GitHub Action to deploy Cloud Native Patterns on CI/CD pipelines.",
-        repository: "https://github.com/layer5io/meshery-service-mesh-patterns-action",
+        maintainers_name: ["Lee Calcote"],
+        link: ["https://layer5.io/community/members/lee-calcote"],
+        description:
+          "GitHub Action to deploy Cloud Native Patterns on CI/CD pipelines.",
+        repository:
+          "https://github.com/layer5io/meshery-service-mesh-patterns-action",
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:448]:
```suggestion
          "https://github.com/layer5io/meshery-cloud-native-patterns-action",
```

File: src/sections/Community/Handbook/repo-data.js [hunk 21]
```diff
@@ -368,8 +440,12 @@ export const repo_data = [
         project: "Cloud Native Patterns Action",
         image: patterns,
         language: "shell",
-        description: "GitHub Action to deploy Cloud Native Patterns on CI/CD pipelines.",
-        repository: "https://github.com/layer5io/meshery-service-mesh-patterns-action",
+        maintainers_name: ["Lee Calcote"],
+        link: ["https://layer5.io/community/members/lee-calcote"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:444]:
```suggestion
        link: ["#"],
```

File: src/sections/Community/Handbook/repo-data.js [hunk 22]
```diff
@@ -368,8 +440,12 @@ export const repo_data = [
         project: "Cloud Native Patterns Action",
         image: patterns,
         language: "shell",
-        description: "GitHub Action to deploy Cloud Native Patterns on CI/CD pipelines.",
-        repository: "https://github.com/layer5io/meshery-service-mesh-patterns-action",
+        maintainers_name: ["Lee Calcote"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:443]:
```suggestion
        maintainers_name: ["Vacant"],
```

File: src/sections/Community/Handbook/repo-data.js [hunk 23]
```diff
@@ -380,110 +456,142 @@ export const repo_data = [
...
+        maintainers_name: ["Hussaina Begum"],
+        link: ["https://layer5.io/community/members/hussaina-begum"],
         description: "Meshery adapter for management of Consul",
         repository: "https://github.com/layer5io/meshery-consul",
       },
       {
         project: "Meshery Adapter for CPX",
         image: meshery,
         language: "Golang, gRPC",
+        maintainers_name: ["Lee Calcote"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:522]:
```suggestion
        maintainers_name: ["Vacant"],
```

File: src/sections/Community/Handbook/repo-data.js [hunk 24]
```diff
@@ -380,110 +456,142 @@ export const repo_data = [
...
+        link: ["https://layer5.io/community/members/lee-calcote"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:523]:
```suggestion
        link: ["#"],
```

File: src/sections/Community/Handbook/repo-data.js [hunk 25]
```diff
@@ -380,110 +456,142 @@ export const repo_data = [
...
       },
       {
         project: "Meshery Adapter for Tanzu Service Mesh",
         image: meshery,
         language: "Golang, gRPC",
-        description: "Meshery adapter for management of VMware Tanzu Service Mesh",
+        maintainers_name: ["Xin Huang"],
+        link: ["https://layer5.io/community/members/xin-huang"],
+        description:
+          "Meshery adapter for management of VMware Tanzu Service Mesh",
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:543]:
```suggestion
        link: ["https://layer5.io/community/members/hussaina-begum"],
        description:
          "Meshery adapter for management of VMware Tanzu Service Mesh",
```

File: src/sections/Community/Handbook/repo-data.js [hunk 26]
```diff
@@ -380,110 +456,142 @@ export const repo_data = [
...
+        link: ["https://layer5.io/community/members/antonette-caldwell"],
         description: "Meshery adapter for management of AWS App Mesh",
         repository: "https://github.com/layer5io/meshery-app-mesh",
       },
       {
         project: "Meshery Adapter for Tanzu Service Mesh",
         image: meshery,
         language: "Golang, gRPC",
-        description: "Meshery adapter for management of VMware Tanzu Service Mesh",
+        maintainers_name: ["Xin Huang"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:540]:
```suggestion
        maintainers_name: ["Hussaina Begum"],
```

File: src/sections/Community/Handbook/repo-data.js [hunk 27]
```diff
@@ -41,26 +41,29 @@ export const repo_data = [
         image: meshery,
         site: "https://meshery.io",
         language: "Jekyll",
-        maintainers_name: ["Vacant"],
-        link: ["#"],
+        maintainers_name: ["Nikhil Ladha"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:44]: Change to Vivek Vishal, please.

**Comment by l5io** [comment]
🚀 Preview for commit 7a1c9b010176dddce9375ea82fcd3911eb40ccb9 at: https://6555a3a243d1113b3204625a--layer5.netlify.app

**Comment by leecalcote** [comment]
Thank you, @ankushbhardwaj408 👏

**Comment by ankushbhardwaj408** [comment]
Thanks to you @leecalcote for your patience and all the suggestions. Happy to contribute!🫡

**Pull request merged**